docs/source/library.rst

```diff
@@ -10,6 +10,6 @@ simtools Library
   mc_model
   ray_tracing
   camera_efficiency
-  simtel_runner
+  simtel
```

Comment:
KB will tell you that it is sim_telarray....

Reply:
We write 'sim_telarray' in the documentation. 'simtel' is just a term used in the code.

Reply:
Always good to be consistent (but it really doesn't matter)